test/integration_test/common_test.go

```diff
@@ -672,8 +693,13 @@ func scheduleBidirectionalClusterPair(cpName, cpNamespace string) error {
 		return fmt.Errorf("during cluster pair setting kubeconfig to source failed %v", err)
 	}
 
+	token, err = getTokenFromSecret(adminTokenSecretName, defaultAdminNamespace)
```

Comment:
No authTokenConfigMap check on this one?

Reply:
Fixed.